app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -39,19 +39,26 @@
 
 .gem-c-feedback__prompt-question,
 .gem-c-feedback__prompt-success {
-  display: inline;
-  font-weight: bold;
+  @include bold-19;
```

Comment:
This was `bold-16` before it was changed in 12c1b283186b639f3b729264f0f17ff2b2121f63. Is it intentional that it's 19px now?

Reply:
I think this is to ensure that the font size is 16px on both desktop and mobile - our type mixins automatically shrink the font for mobile, which is a bit unreadable if you start at 16px on desktop.